test/collection.js

```diff
@@ -701,4 +701,18 @@ describe('Chained Collection', () => {
       await Collect([4, 1]).avg()
     ).to.equal(2.5)
   })
+
+  it('median', async () => {
+    expect(
+      await Collect([1, 2, 3]).median()
+    ).to.equal(2)
+
+    expect(
+      await Collect([1, 3, 2]).median()
```

Comment:
Sweet! I like this test to ensure the sorting before calculating the median :+1:

Reply:
Thanks